tools/py/Instanton_postproc.py

```diff
@@ -393,11 +393,11 @@ def get_rp_freq(w0, nbeads, temp, mode="rate"):
     else:
         Qrot = 1.0
 
-    outfile = open("freq.dat", "w")
-    aux = np.zeros(nzeros)
-    dd = np.concatenate((aux, d))
-    np.savetxt(outfile, dd.reshape(1, dd.size))
-    outfile.close()
+    # outfile = open("freq.dat", "w")
+    # aux = np.zeros(nzeros)
+    # dd = np.concatenate((aux, d))
+    # np.savetxt(outfile, dd.reshape(1, dd.size))
+    # outfile.close()
```

Comment:
Could we delete these commented lines?

Reply:
P.S.: I think it is a good idea to print or save the actual frequencies (not eigenvalues) somewhere. Is it too verbose to make it an actual output of the script?

Reply:
I created a simple save_frequencies function and deleted the useless commented lines.